### ISSUE TITLE
Benchmark service should only notify the Slack channel once per day

### ISSUE THREAD

**Issue by lemmih** [ChainSafe/forest-iac#247]
**Issue summary**

The benchmark service notifies the slack channel after each successful run. This should be changed to only notify once per day.

**Other information and links**

**Comment by lemmih** [comment]
> @lemmih just to clarify, should it just run once per day or find a logic for it to notify only once per day but still run continuously as it is now

It should run continuously but only report to Slack once per day (or even less frequently than that).

**Comment by LesnyRumcajs** [comment]
How about a summary of benchmark runs from the past day? It would nicely show how results changed, and we could act on this if they get weird.

**Comment by lemmih** [comment]
Including useful information in the status report would be a nice feature but it is out of scope for this issue. This issue is merely about not spamming the Slack channel.

**Comment by LesnyRumcajs** [comment]
@samuelarogbonlo to make it easy, let's just follow the daily snapshot steps.

https://github.com/ChainSafe/forest-iac/blob/f3b5b418b3725a2f463026ad71592750b98effea/terraform/modules/daily_snapshot/service/daily_snapshot.rb#L41-L44

Once per day for now is fine. Later we can increase the interval. Successful operation definition remains the same as for normal run - the process exits with code 0. That is to say, we care only about the benchmark run at hand.

**Comment by lemmih** [comment]
> @lemmih okay i understand that, But what I'm still trying to grasp is: to prevent spamming, should the success message be sent only once a day or week? And what, in your opinion, should define a successful operation since the benchmark runs consistently? That part is a bit unclear to me.

The current service sends a message to #forest-notifications after every successful run and after every failed run. We _only_ want to change how often success messages are sent to the Slack channel. This does not require re-defining what a successful run is.
